---
layout: page
---
# bulkATACseq

## Overview:
This schema is for Assay for Transposase-Accessible Chromatin by sequencing (ATACseq)of bulk sample.

### Last Updated: 2023-04-25

### Version 1 (current)

## Metadata Schema

| Attribute | Description | shared/unique | enum | required | format | required if | type | pattern (regular expression) | url |
|-----------|-------------|---------------|------|----------|--------|-------------|------|------------------------------|-----|
| version | Version of the schema to use when validating this metadata. | Shared by all types | 1 | True |   |   |   |   |   |
| description | Free-text description of this assay. | Shared by all types |   | True |   |   |   |   |   |
| donor_id | HuBMAP Display ID of the donor of the assayed tissue. Example: ABC123. | Shared by all types |   | True |   |   |   | [A-Z]+[0-9]+ |   |
| tissue_id | HuBMAP Display ID of the assayed tissue. Example: ABC123-BL-1-2-3_456. | Shared by all types |   | True |   |   |   | (([A-Z]+[0-9]+)-[A-Z]{2}\d*(-\d+)+(_\d+)?)(,([A-Z]+[0-9]+)-[A-Z]{2}\d*(-\d+)+(_\d+)?)* |   |
| execution_datetime | Start date and time of assay, typically a date-time stamped folder generated by the acquisition instrument. YYYY-MM-DD hh:mm, where YYYY is the year, MM is the month with leading 0s, and DD is the day with leading 0s, hh is the hour with leading zeros, mm are the minutes with leading zeros. | Shared by all types |   | True | %Y-%m-%d %H:%M |   | datetime |   |   |
| protocols_io_doi | DOI for protocols.io referring to the protocol for this assay. | Shared by all types |   | True |   |   |   | 10\.17504/.* | prefix: https://dx.doi.org/ |
| operator | Name of the person responsible for executing the assay. | Shared by all types |   | True |   |   |   |   |   |
| operator_email | Email address for the operator. | Shared by all types |   | True | email |   |   |   |   |
| pi | Name of the principal investigator responsible for the data. | Shared by all types |   | True |   |   |   |   |   |
| pi_email | Email address for the principal investigator. | Shared by all types |   | True | email |   |   |   |   |
| assay_category | Each assay is placed into one of the following 4 general categories: generation of images of microscopic entities, identification & quantitation of molecules by mass spectrometry, imaging mass spectrometry, and determination of nucleotide sequence. | Shared by all types | sequence | True |   |   |   |   |   |
| assay_type | The specific type of assay being executed. | Shared by all types | bulkATACseq | True |   |   |   |   |   |
| analyte_class | Analytes are the target molecules being measured with the assay. | Shared by all types | DNA | True |   |   |   |   |   |
| is_targeted | Specifies whether or not a specific molecule(s) is/are targeted for detection/measurement by the assay. | Shared by all types |   | True |   |   | boolean |   |   |
| acquisition_instrument_vendor | An acquisition instrument is the device that contains the signal detection hardware and signal processing software. Assays generate signals such as light of various intensities or color or signals representing the molecular mass. | Shared by all types |   | True |   |   |   |   |   |
| acquisition_instrument_model | Manufacturers of an acquisition instrument may offer various versions (models) of that instrument with different features or sensitivities. Differences in features or sensitivities may be relevant to processing or interpretation of the data. | Shared by all types |   | True |   |   |   |   |   |
| bulk_transposition_input_number_nuclei | A number (no comma separators) | Unique to this type |   | True |   |   |   |   |   |
| bulk_atac_cell_isolation_protocols_io_doi | Link to a protocols document answering the question: How was tissue stored and processed for cell/nuclei isolation. | Unique to this type |   | True |   |   |   | 10\.17504/.* | prefix: https://dx.doi.org/ |
| is_technical_replicate | Is this a sequencing replicate? | Unique to this type |   | True |   |   | boolean |   |   |
| library_adapter_sequence | Adapter sequence to be used for adapter trimming. | Unique to this type |   | True |   |   |   |   |   |
| library_average_fragment_size | Average size in basepairs (bp) of sequencing library fragments estimated via gel electrophoresis or bioanalyzer/tapestation. | Unique to this type |   | True |   |   | integer |   |   |
| library_concentration_value | The concentration value of the pooled library samples submitted for sequencing. | Unique to this type |   | True |   |   | number |   |   |
| library_concentration_unit | Unit of library_concentration_value. Leave blank if not applicable. | Unique to this type | nM | False |   | library_concentration_value present |   |   |   |
| library_construction_protocols_io_doi | A link to the protocol document containing the library construction method (including version) that was used, e.g. “Smart-Seq2”, “Drop-Seq”, “10X v3”. | Unique to this type |   | True |   |   |   | 10\.17504/.* | prefix: https://dx.doi.org/ |
| library_creation_date | date and time of library creation. YYYY-MM-DD, where YYYY is the year, MM is the month with leading 0s, and DD is the day with leading 0s. Leave blank if not applicable. | Unique to this type |   | False | %Y-%m-%d |   | datetime |   |   |
| library_final_yield_value | Total number of ng of library after final pcr amplification step. This is the concentration (ng/ul) * volume (ul) | Unique to this type |   | True |   |   | number |   |   |
| library_final_yield_unit | Units of final library yield. Leave blank if not applicable. | Unique to this type | ng | False |   | library_final_yield_value present |   |   |   |
| library_id | A library ID, unique within a TMC, which allows corresponding RNA and chromatin accessibility datasets to be linked. | Unique to this type |   | True |   |   |   |   |   |
| library_layout | State whether the library was generated for single-end or paired end sequencing. | Unique to this type | single-end or paired-end | True |   |   |   |   |   |
| library_pcr_cycles | Number of PCR cycles performed in order to add adapters and amplify the library. Usually, this includes 5 pre-amplificationn cycles followed by 0-5 additional cycles determined by qPCR. | Unique to this type |   | True |   |   | integer |   |   |
| library_preparation_kit | Reagent kit used for library preparation. | Unique to this type |   | True |   |   |   |   |   |
| sample_quality_metric | This is a quality metric by visual inspection. This should answer the question: Are the nuclei intact and are the nuclei free of significant amounts of debris? This can be captured at a high level, “OK” or “not OK”. | Unique to this type |   | True |   |   |   |   |   |
| sequencing_phix_percent | Percent PhiX loaded to the run. | Unique to this type |   | True |   |   | number |   |   |
| sequencing_read_format | Slash-delimited list of the number of sequencing cycles for, for example, Read1, i7 index, i5 index, and Read2. Example: 12/34/56. | Unique to this type |   | True |   |   |   | \d+(/\d+)+ |   |
| sequencing_read_percent_q30 | Q30 is the weighted average of all the reads (e.g. # bases UMI * q30 UMI + # bases R2 * q30 R2 + …) | Unique to this type |   | True |   |   | number |   |   |
| sequencing_reagent_kit | Reagent kit used for sequencing. | Unique to this type |   | True |   |   |   |   |   |
| transposition_kit_number | If Tn5 came from a kit, provide the catalog number. Leave blank if not applicable. | Unique to this type |   | False |   |   |   |   |   |
| transposition_method | Modality of capturing accessible chromatin molecules. The kit used, for example. | Unique to this type |   | True |   |   |   |   |   |
| transposition_transposase_source | The source of the Tn5 transposase and transposon used for capturing accessible chromatin. | Unique to this type |   | True |   |   |   |   |   |
| contributors_path | Relative path to file with ORCID IDs for contributors for this dataset. | Unique to this type |   | True |   |   |   |   |   |
| data_path | Relative path to file or directory with instrument data. Downstream processing will depend on filename extension conventions. | Unique to this type |   | True |   |   |   |   |   |
