---
layout: page
---
# SNARE-seq2 / sciATACseq / snATACseq

## Overview:
This schema is for the single nucleus Assay for Transposase Accessible Chromatin by sequencing (snATACseq).

### Last Updated: 2023-04-25

### Version 1 (current)

## Metadata Schema

| Attribute | Description | shared/unique | enum | required | format | required if | type | pattern (regular expression) | url |
|-----------|-------------|---------------|------|----------|--------|-------------|------|------------------------------|-----|
| version | Version of the schema to use when validating this metadata. | Shared by all types | 1 | True |   |   |   |   |   |
| description | Free-text description of this assay. | Shared by all types |   | True |   |   |   |   |   |
| donor_id | HuBMAP Display ID of the donor of the assayed tissue. Example: ABC123. | Shared by all types |   | True |   |   |   | [A-Z]+[0-9]+ |   |
| tissue_id | HuBMAP Display ID of the assayed tissue. Example: ABC123-BL-1-2-3_456. | Shared by all types |   | True |   |   |   | (([A-Z]+[0-9]+)-[A-Z]{2}\d*(-\d+)+(_\d+)?)(,([A-Z]+[0-9]+)-[A-Z]{2}\d*(-\d+)+(_\d+)?)* |   |
| execution_datetime | Start date and time of assay, typically a date-time stamped folder generated by the acquisition instrument. YYYY-MM-DD hh:mm, where YYYY is the year, MM is the month with leading 0s, and DD is the day with leading 0s, hh is the hour with leading zeros, mm are the minutes with leading zeros. | Shared by all types |   | True | %Y-%m-%d %H:%M |   | datetime |   |   |
| protocols_io_doi | DOI for protocols.io referring to the protocol for this assay. | Shared by all types |   | True |   |   |   | 10\.17504/.* | prefix: https://dx.doi.org/ |
| operator | Name of the person responsible for executing the assay. | Shared by all types |   | True |   |   |   |   |   |
| operator_email | Email address for the operator. | Shared by all types |   | True | email |   |   |   |   |
| pi | Name of the principal investigator responsible for the data. | Shared by all types |   | True |   |   |   |   |   |
| pi_email | Email address for the principal investigator. | Shared by all types |   | True | email |   |   |   |   |
| assay_category | Each assay is placed into one of the following 4 general categories: generation of images of microscopic entities, identification & quantitation of molecules by mass spectrometry, imaging mass spectrometry, and determination of nucleotide sequence. | Shared by all types | sequence | True |   |   |   |   |   |
| assay_type | The specific type of assay being executed. | Shared by all types | SNARE-seq2, sciATACseq, or snATACseq | True |   |   |   |   |   |
| analyte_class | Analytes are the target molecules being measured with the assay. | Shared by all types | DNA | True |   |   |   |   |   |
| is_targeted | Specifies whether or not a specific molecule(s) is/are targeted for detection/measurement by the assay. | Shared by all types |   | True |   |   | boolean |   |   |
| acquisition_instrument_vendor | An acquisition instrument is the device that contains the signal detection hardware and signal processing software. Assays generate signals such as light of various intensities or color or signals representing the molecular mass. | Shared by all types |   | True |   |   |   |   |   |
| acquisition_instrument_model | Manufacturers of an acquisition instrument may offer various versions (models) of that instrument with different features or sensitivities. Differences in features or sensitivities may be relevant to processing or interpretation of the data. | Shared by all types |   | True |   |   |   |   |   |
| is_technical_replicate | If TRUE, fastq files in dataset need to be merged. | Unique to this type |   | True |   |   | boolean |   |   |
| library_id | A library ID, unique within a TMC, which allows corresponding RNA and chromatin accessibility datasets to be linked. | Unique to this type |   | True |   |   |   |   |   |
| sc_isolation_protocols_io_doi | Link to a protocols document answering the question: How were single cells separated into a single-cell suspension? | Unique to this type |   | True |   |   |   | 10\.17504/.* | prefix: https://dx.doi.org/ |
| sc_isolation_entity | The type of single cell entity derived from isolation protocol. | Unique to this type | whole cell, nucleus, cell-cell multimer, or spatially encoded cell barcoding | True |   |   |   |   |   |
| sc_isolation_tissue_dissociation | The method by which tissues are dissociated into single cells in suspension. | Unique to this type |   | True |   |   |   |   |   |
| sc_isolation_enrichment | The method by which specific cell populations are sorted or enriched. | Unique to this type | none or FACS | True |   |   |   |   |   |
| sc_isolation_quality_metric | A quality metric by visual inspection prior to cell lysis or defined by known parameters such as wells with several cells or no cells. This can be captured at a high level. “OK” or “not OK”, or with more specificity such as “debris”, “clump”, “low clump”. | Unique to this type |   | True |   |   |   |   |   |
| sc_isolation_cell_number | Total number of cell/nuclei yielded post dissociation and enrichment. | Unique to this type |   | True |   |   | integer |   |   |
| transposition_input | Number of cell/nuclei input to the assay. | Unique to this type |   | True |   |   | number |   |   |
| transposition_method | Modality of capturing accessible chromatin molecules. | Unique to this type | SNARE-Seq2-AC, bulkATACseq, snATACseq, or sciATACseq | True |   |   |   |   |   |
| transposition_transposase_source | The source of the Tn5 transposase and transposon used for capturing accessible chromatin. | Unique to this type | 10X snATAC, In-house, Nextera, or 10X multiome | True |   |   |   |   |   |
| transposition_kit_number | If Tn5 came from a kit, provide the catalog number. Leave blank if not applicable. | Unique to this type |   | False |   |   |   |   |   |
| library_construction_protocols_io_doi | A link to the protocol document containing the library construction method (including version) that was used, e.g. “Smart-Seq2”, “Drop-Seq”, “10X v3”. DOI for protocols.io referring to the protocol for this assay. | Unique to this type |   | True |   |   |   | 10\.17504/.* | prefix: https://dx.doi.org/ |
| library_layout | State whether the library was generated for single-end or paired end sequencing. | Unique to this type | single-end or paired-end | True |   |   |   |   |   |
| library_adapter_sequence | Adapter sequence to be used for adapter trimming. Example: CTGTCTCTTATACACATCT. | Unique to this type |   | True |   |   |   | [ATCG]+(\+[ATCG]+)? |   |
| cell_barcode_read | Which read file(s) contains the cell barcode. Multiple cell_barcode_read files must be provided as a comma-delimited list (e.g. file1,file2,file3). This field is not required for barcoding by single-cell combinatorial indexing. Leave blank if not applicable. | Unique to this type |   | False |   |   |   |   |   |
| cell_barcode_offset | Positions in the read at which the cell barcodes start. Cell barcodes are, for example, 3 x 8 bp sequences that are spaced by constant sequences (the offsets). First barcode at position 0, then 38, then 76. (Does not apply to sciATACseq, SNARE-seq and BulkATAC.) Leave blank if not applicable. Example: 0,0,38,76. | Unique to this type |   | False |   |   |   | \d+(,\d+)* |   |
| cell_barcode_size | Length of the cell barcode in base pairs. Cell barcodes are, for example, 3 x 8 bp sequences that are spaced by constant sequences, the offsets. (Does not apply to sciATACseq, SNARE-seq and BulkATAC.) Leave blank if not applicable. Example: 16,8,8,8. | Unique to this type |   | False |   |   |   | \d+(,\d+)* |   |
| library_pcr_cycles | Number of PCR cycles to enrich for accessible chromatin fragments. | Unique to this type |   | True |   |   | integer |   |   |
| library_pcr_cycles_for_sample_index | Number of PCR cycles performed for library generation (figure in Descriptions section) | Unique to this type |   | True |   |   | integer |   |   |
| library_final_yield | Total ng of library after final pcr amplification step. | Unique to this type |   | True |   |   | number |   |   |
| library_final_yield_unit | Units for library_final_yield. Leave blank if not applicable. | Unique to this type | ng | False |   | library_final_yield present |   |   |   |
| library_average_fragment_size | Average size in basepairs (bp) of sequencing library fragments estimated via gel electrophoresis or bioanalyzer/tapestation. | Unique to this type |   | True |   |   | integer |   |   |
| sequencing_reagent_kit | Reagent kit used for sequencing. | Unique to this type |   | True |   |   |   |   |   |
| sequencing_read_format | Slash-delimited list of the number of sequencing cycles for, for example, Read1, i7 index, i5 index, and Read2. Example: 12/34/56. | Unique to this type |   | True |   |   |   | \d+(/\d+)+ |   |
| sequencing_read_percent_q30 | Q30 is the weighted average of all the reads (e.g. # bases UMI * q30 UMI + # bases R2 * q30 R2 + …) | Unique to this type |   | True |   |   | number |   |   |
| sequencing_phix_percent | Percent PhiX loaded to the run. | Unique to this type |   | True |   |   | number |   |   |
| contributors_path | Relative path to file with ORCID IDs for contributors for this dataset. | Unique to this type |   | True |   |   |   |   |   |
| data_path | Relative path to file or directory with instrument data. Downstream processing will depend on filename extension conventions. | Unique to this type |   | True |   |   |   |   |   |
