---
layout: page
---
# HuBMAP Frequently Asked Questions (FAQ)

## General Questions

<details>
<summary><b>What is HuBMAP & what are its goals?</b></summary>

HuBMAP is the Human BioMolecular Atlas Program.
The vision for HuBMAP is to catalyze the development of a framework for mapping of the human body at single-cell resolution to transform our understanding of normal tissue organization and function. This will be achieved by:

<ul>
    <li>Accelerating the development of the next generation of tools and techniques for constructing high resolution spatial tissue maps that quantify multiple types of biomolecules either sequentially or simultaneously.</li>
    <li>Generating foundational 3D human tissue maps using validated high-content, high-throughput imaging and omics assays.</li>
    <li>Establishing an open data platform that will develop novel approaches to integrating, visualizing, and modelling imaging and omics data to build multi-dimensional tissue maps, and making data rapidly findable, accessible, interoperable, and reusable by the global research community.</li>
    <li>Coordinating and collaborating with other funding agencies, programs, and the biomedical research community to build the framework and tools for mapping the human body at single-cell resolution.</li>
    <li>Supporting pilot projects that demonstrate the value of the resources developed by the program to study normal individual variations and tissue changes across the lifespan and the health-disease continuum.</li>
</ul>

Better insights into the principles governing the tissue organization-function relationship will:
<ul>
   <li>Potentially lead to better understanding of the significance of normal inter-individual variability and changes across the lifespan.</li>
   <li>Inform about the emergence of disease at the biomolecular level before the appearance of clinical symptoms.</li>
</ul>

Despite vastly improved imaging and omics technologies and many important foundational discoveries, our understanding of how tissues are organized is still restricted by main challenges: 

<ol>
   <li>Integrating high content, high resolution spatial and omics information to comprehensively profile biomolecular distribution and morphology of tissues in a high throughput manner.</li>
   <li>Arranging this information into 3D tissue maps amenable to modelling.</li>
</ol>

Generating foundational 3D human tissue maps is one of the core goals of HuBMAP. HuBMAP projects will generate _high resolution, high content, high-throughput_ biomolecular 3D tissue maps of non-diseased human organs and organ systems. 

<ul>
    <li>For HuBMAP, a <em>high-resolution</em> assay is one that can reliably and reproducibly assign detected biomolecules to individual cells or extracellular compartments of a tissue.</li>
    <li>A <em>high content</em> approach is one that maximizes identification of tissue features through a combination of biomolecular depth, spatial resolution and multiplexing of complementary, multi-parameter assays.</li>
    <li>A <em>high throughput</em> pipeline is one that maximizes the bandwidth of data production to result in any or all of the following:</li>
</ul>
    <ol>
       <li>Accelerated speed of analysis, so that hundreds or thousands of samples can be analyzed simultaneously.</li>
       <li>Greater depth of analysis, so that hundreds or thousands of molecules can be analyzed in a single sample.</li>
       <li>Enhanced capacity for volume, so that a given set of molecules can be analyzed in all the cells within a larger tissue sample.</li>
    </ol>

Using a multi-dimensional approach, including imaging, sequencing, and mass spectrometry assays, HuBMAP provides robust molecular characterization of human cells in their natural tissue context. HuBMAP also generates and shares a number of other resources to support the use of these maps, including details of experimental protocols used, validation of affinity probes, biospecimen metadata, conventions used for annotation, as well as computational tools.

<p><a href="https://commonfund.nih.gov/hubmap">HuBMAP</a>, which made its first external awards in Fall 2018, is funded through the NIH Common Fund as a short-term (8 years), goal-driven strategic investment, with deliverables intended to catalyze research across multiple biomedical research disciplines. The <a href="https://commonfund.nih.gov/about">NIH Common Fund</a> supports cross-cutting programs that are expected to have exceptionally high impact. <a href="https://commonfund.nih.gov/programs">All Common Fund</a> initiatives invite investigators to develop bold, innovative, and often risky approaches to address problems that may seem intractable in isolation or to seize new opportunities that offer the potential for rapid progress.

<p>For a more in depth understanding, read the <a href="https://www.nature.com/articles/s41586-019-1629-x">HuBMAP marker paper</a>, see the course on HuBMAP data acquisition, analysis, and visualization, the <a href="https://expand.iu.edu/browse/sice/cns/courses/hubmap-visible-human-mooc">Visible Human MOOC</a>, or see this video: <a href="https://www.youtube.com/watch?v=yCh4XnD7rEE">HuBMAP Overview</a>. Stay in touch by subscribing to our <a href="https://hubmapconsortium.org/hubmap-mailing-list">mailing list</a> and <a href="https://www.youtube.com/channel/UCbSvPJ9dXASL14KoDeutMFg">YouTube channel</a>.
</details>

<details><summary><b>Who are the intended users of HuBMAP resources?</b></summary>

HuBMAP's rich datasets and associated resources are intended for broad use by the research community, including:
<ul>
    <li>Computational researchers exploring organizing principles of human tissues, new structural-functional relationships, and biomolecular networks </li>
    <li>Biologists exploring hypotheses using publicly available HuBMAP datasets prior to or in parallel with work in their own labs</li>
    <li>Experimentalists interested in using the same protocols or computational tools in their labs </li>
    <li>Educators developing new teaching materials</li>
    <li>Technology developers interested in developing new assays with enhanced performance </li>
</ul>
</details>

<details><summary><b>How does HuBMAP fit into the ecosystem of other single cell efforts within and outside the NIH?</b> </summary>

HuBMAP is part of a rich ecosystem of established and emerging atlasing programs supported by NIH and globally by other funding organizations, many of which are focused on specific organs or diseases. 
<ul>
    <li>HuBMAP connects with these programs to ensure data interoperability, avoid duplication of work, and leverage and synergize gained knowledge.</li>
    <li>The consortium has organized a number of events to bring together these communities to discuss topics of shared interest (e.g. <a href="https://hubmapconsortium.org/nihhca2020/">CCF meeting, NIH-HCA meeting</a>) and is committed to improving coordination and collaboration among different programs.</li>
    <li>In addition, many HuBMAP PIs actively participate in these efforts, helping with cross-pollination and advancing our global understanding. </li>
</ul>

HuBMAP, as its name implies, was specifically initiated to resolve the challenge of building integrated, comprehensive, high-resolution spatial maps of human tissues and organs. HuBMAP provides leadership in the ecosystem around:
<ul>
    <li>Techniques for integrating disparate, multi-dimensional and multi-scale datasets</li>
    <li>The development of a Common Coordinate Framework (CCF) for integrating data across many individuals</li>
    <li>The development and validation of these assays</li>
</ul>

To further increase interoperability, HuBMAP has adopted a number of standards and processes developed by other consortia, and is working and actively involved in knowledge exchange. The consortium sees itself as an integral part of the ecosystem, sharing its strengths and actively contributing to the community.

</details>

## Introduction to HuBMAP resources

We welcome all to explore the <a href="https://hubmapconsortium.org/">HuBMAP Consortium website</a> and <a href="https://portal.hubmapconsortium.org/">HuBMAP Data Portal</a>. 

Visitors can provide feedback through the <a href="https://hubmapconsortium.org/contact-form/">Contact form</a>.

The <a href="https://expand.iu.edu/browse/sice/cns/courses/hubmap-visible-human-mooc">Visible Human MOOC</a> provides an overview of HuBMAP and an introduction to data acquisition, analysis, and visualization.

## Getting Started with the HuBMAP Data Portal

<details><summary><b>How can I start using the portal?</b> </summary>

Here's a link to the <a href="https://portal.hubmapconsortium.org/">HuBMAP Data Portal</a>. HuBMAP Data Portal (and related) documentation can be found on the <a href="https://software.docs.hubmapconsortium.org/technical">HuBMAP Documentation</a> page and also the <a href="https://software.docs.hubmapconsortium.org/faq">HuBMAP FAQ</a> - <em>this</em> page.
 
</details>

<details><summary><b>How can I learn more about HuBMAP data acquisition, analysis, and visualization tools?</b> </summary>

The <a href="https://expand.iu.edu/browse/sice/cns/courses/hubmap-visible-human-mooc">Visible Human MOOC</a> provides an overview of HuBMAP and introduction to data acquisition, analysis, and visualization. 

</details>

<details><summary><b>Can I sign up to be notified of HuBMAP news and events?</b> </summary>

Yes, you can <a href="https://hubmapconsortium.org/hubmap-mailing-list/">sign up for our mailing list</a> to keep informed on everything that is happening in HuBMAP.

</details>

<details><summary><b>Do you track my activities on your portal?</b> </summary>

Yes, interactions with the site are recorded in server logs and on Google Analytics and are mapped to your IP address. 
<br />In that regard the HuBMAP portal is no different from the rest of the internet.

</details>

<details><summary><b>What web-browsers are supported in the portal?</b> </summary>

All modern, mainstream browsers are supported (i.e. Chrome, Edge, Firefox, Safari, etc.).

</details>

<details><summary><b>Where should I report errors, make suggestions for new features or functionality, or ask questions about using the portal?</b> </summary>

The HuBMAP Consortium welcomes your comments, feedback, and help in identifing errors on the HuBMAP Data Portal. 
<ul>
    <li>You can provide error reports, make suggestions, or ask questions through our <a href="https://hubmapconsortium.org/contact-form/">contact-form</a>.</li>
    <li>For help with specific issues related to the portal, please contact the <a href="mailto:help@hubmapconsortium.org">HuBMAP Helpdesk</a> and submit a support ticket.</li>
</ul>

</details>

## HuBMAP Data and Metadata Questions

<details><summary><b>Can I use HuBMAP data for my own research?</b> </summary>

<ul>
    <li>Yes, follow the guidelines outlined in the <a href="https://hubmapconsortium.org/policies/external-data-sharing-policy/">HuBMAP External Data Sharing Policy</a>. </li>
    <li>Access to NIH HuBMAP data is guided by the <a href="https://sharing.nih.gov/faqs#/genomic-data-sharing-policy.htm">NIH Genomic Data Sharing policy</a>.</li>
</ul>

If you use NIH HuBMAP data in publications or presentations we request that you include an acknowledgement of the HuBMAP Program.
<ul>
    <li>This acknowledgement helps justify and sustain funding needed to continue providing open access to a growing set of data and tools.</li>
    <li>Suggested language for such an acknowledgment is: “The results (published or shown) here are in whole or part based upon data generated by the <a href="https://hubmapconsortium.org">HuBMAP Program</a>."</li>
</ul>
        
</details>

<details><summary><b>Can I use your data for another project that I am working on?</b></summary>

Yes! The Consortium provides raw and processed data (at multiple levels) for the community to access through the <a href="https://portal.hubmapconsortium.org/">HuBMAP Data Portal</a>. <br />

HuBMAP products are broadly available to the research community to establish the foundations for a human body map that other programs and the international community can build upon, including methods, tools, reagents, biospecimens, datasets, and software. <br />

To acknowledge HuBMAP in your findings, the Consortium suggests language of the form: “The results (published or shown) here are in whole or part based upon data generated by the <a href="https://hubmapconsortium.org">HuBMAP Program</a>."

</details>

<details><summary><b>Can I submit my own data to HuBMAP? Why would I want to?</b></summary>

<ul>
    <li>Yes, HuBMAP allows investigators to submit their own data via the <a href="https://portal.hubmapconsortium.org/">HuBMAP Data Portal</a>.</li>
    <li>Why share? Having your own data on HuBMAP will ...</li>
    <ul>
       <li>Allow other researchers access to your results. In this way, others may extend and interact with your scientific work.</li>
       <li>Provide additional resources for creating cellular and molecular level anatomical maps of the healthy human.</li>
    </ul>
</ul>
The HuBMAP consortium encourages the scientific community to provide feedback about HuBMAP dataset metadata. This feedback helps improve the quality and usability of community data submissions.

</details>

<details><summary><b>Can I add my own tools to HuBMAP portal? Why would I want to submit my tools to HuBMAP?</b></summary>

<ul>
    <li>Yes, HuBMAP seeks to host relevant tools and welcomes community input to help with feature prioritization and development for the HuBMAP Portal.</li>
    <li>Adding your Tools to the HuBMAP Portal will help you get others to use your tools and provide feedback to improve the scientific impact of your work.</li>
    <li>One of the first tools released was <a href="https://azimuth.hubmapconsortium.org/">Azimuth</a>, an app for reference-based single-cell analysis, that lets users annotate cell-types in their own data based on HuBMAP approaches. </li>
</ul>
</details>

<details><summary><b>What do I need to do to gain access to HuBMAP data? How can I download HuBMAP data?</b> </summary>

Access to data on [HuBMAP's Data Portal](https://portal.hubmapconsortium.org/) is open to all interested viewers, without additional barriers (account creation, login, etc.).

<b>How can I download HuBMAP data?</b> <br>
To download HuBMAP data from the Data Portal you will need to register as a member of HuBMAP. Note that downloads of specific datasets (e.g., raw genetic data) require NIH approval. Contact the [HuBMAP Helpdesk](mailto:help@hubmapconsortium.org) for assitance with accessing this type of data.

<b>How can I register as a member of HuBMAP?</b> <br>
    1. Go to the [HuBMAP Consortium website](https://hubmapconsortium.org/) <br>
    2. Under <em>Member Services</em> select <b>Member Register</b> <br>
    3. Complete the registration form (you will need a valid <em>institutional</em> or eRA Commons ID)
    
</details>

<details><summary><b>Can I make HuBMAP data available through my resource?</b> </summary>

<ul>
   <li>You may use HuBMAP data for any purposes permitted by the <a href="https://hubmapconsortium.org/policies/external-data-sharing-policy/">Data Sharing Policy</a>.</li>
   <li>The <a href="https://hubmapconsortium.github.io/ccf/pages/ccf-3d-reference-library.html">CCF 3D Reference Object Library</a> provides anatomically correct reference organs. The organs are developed by a specialist in 3D medical illustration and approved by organ experts.</li> 
   <li>As of December 2023, the 6th HRA release included 30 organ objects that can be freely used in teaching, research, or commercial applications.</li>
</ul>

</details>

<details><summary><b>How would I cite and reference HuBMAP and related data?</b> </summary>

To acknowledge HuBMAP data in publications or presentations, we suggest: 
- “The results \<published or shown\> here are in whole or part based upon data generated by the HuBMAP Program: https://hubmapconsortium.org."
  
The HuBMAP marker paper should be cited as:
- Snyder, M.P., Lin, S., Posgai, A. et al. The human body at cellular resolution: the NIH Human Biomolecular Atlas Program. Nature 574, 187–192 (2019). https://doi.org/10.1038/s41586-019-1629-x.

The Visible Human reference organs are freely available via the CCF 3D Reference Object Library. Please cite as:
- Browne K, Cross LE, Herr, II BW, Record EG, Quardokus EM, Bueckle A, Börner K. 2020. [HuBMAP CCF 3D Reference Object Library.](https://hubmapconsortium.github.io/ccf/pages/ccf-3d-reference-library.html) 

</details>

<details><summary><b>Can I use figures from HuBMAP in my publications?</b> </summary>

Yes, as long as you cite the source of the figure. See the preceeding question for more details.

</details>

<details><summary><b>What metadata is included in HuBMAP? Will HuBMAP include additional metadata in future releases?</b></summary>

- HuBMAP accepts donor, sample (block, section, or suspension), and assay metadata.
   - See this HuBMAP [metadata resource](https://software.docs.hubmapconsortium.org/metadata) for more information.
- In future releases, metadata will be linked to various ontologies to make integration more efficient.

</details>

<details><summary><b>How does HuBMAP curate and validate metadata?</b> </summary>

HuBMAP data submitters can access lists of metadata schemas (and related information) and download metadata templates from HuBMAP’s <a href="https://hubmapconsortium.github.io/ingest-validation-tools/">Data Upload Guidelines</a> page. Data submitters can validate their own metadata using a Validator tool.
<ul>
    <li> Click on any assay or sample type on the lists to jump to a page for that type. </li> 
    <li> Select the <b>Excel template</b> on that page.</li>
    <li> Download and complete the Excel template. </li>
    <li> Validate your metadata spreadsheet (template) using the <a href="https://metadatavalidator.metadatacenter.org/"> Metadata Spreadsheet Validator</a>.</li> 
    <ul>
       <li> The Validator tool compares entries in your metadata template against specifications stored in the CEDAR repository.</li>
       <li> It categorizes any errors found and provides hints on how to fix those errors. </li>
       <li> If errors are detected, use the Validator tools to correct any errors. </li>
       <li> When no errors are found, you may safely upload your spreadsheet.</li>
       <li> Once all errors are corrected, download the <em>corrected</em> metadata spreadsheet (TSV format). </li>
       <li> Send validated (organ or sample) metadata TSV files to the <a href="mailto:help@hubmapconsortium.org">HuBMAP Helpdesk</a> </li>
       <li> A Data Curator at the HuBMAP Helpdesk will manually validate and upload the files.</li>
    </ul>
    <li>Access <a href="https://metadatacenter.github.io/spreadsheet-validator-docs/">Help documentation</a> for the Validator. </li>
    <li> TMCs are required to validate their metadata BEFORE uploading it to the HuBMAP Helpdesk.</li>
</ul>

</details>

<details><summary><b>Is there a “map” in HuBMAP?</b> </summary>

The [Human Reference Atlas (HRA)](https://humanatlas.io) is a comprehensive, high-resolution, three-dimensional atlas of all the cells in a healthy human body. 
- The HRA provides standard terminologies and data structures for describing: 
   - specimens 
   - biological structures
   - spatial positions 

All linked to existing ontologies. Watch a short [video introduction](https://www.youtube.com/watch?v=DDmP_7vDy-o) to the HRA.

- Tools in the HRA include: 
   - Anatomical Structures, Cell Types and Biomarkers [(ASCT+B) Reporter](https://humanatlas.io/asctb-reporter)
   - [Cell Population Graphs](https://humanatlas.io/cell-population-graphs) 
   - [Registration User Interface (RUI)](https://humanatlas.io/registration-user-interface): Register and annotate organs
   - [Exploration User Interface (EUI)](https://humanatlas.io/exploration-user-interface): Interact with registered organs
   - [VR Organ Gallery](https://humanatlas.io/organ-gallery-in-vr): Immersive experience for exploring organs
   - [Millitome](https://humanatlas.io/millitome): 3D-printed tool for organ sectioning
   - [APIs](https://humanatlas.io/api) for querying and interacting with the HRA

</details>

<details><summary><b>How can I run queries? What type queries are currently supported? Can you show me example queries?</b> </summary>

You can query and interact with HuBMAP (and other) data registered in the Human Reference Atlas (HRA) using APIs. 
- [Learn more about HuBMAP HRA APIs](https://humanatlas.io/api)

</details>

<details><summary><b>Can I suggest types of queries I am interested in?</b> </summary>

Contact the [HRA Team](mailto:infoccf@indiana.edu) with feedback or suggestions regarding APIs and queries for HuBMAP HRA data.

</details>

<details><summary><b>What controlled vocabularies or ontologies are being used by HuBMAP?</b> </summary>

Each donor metadata item uses [Unified Medical Language System (UMLS)](https://www.nlm.nih.gov/research/umls/index.html) Concept Unique Identifiers (CUIs) and related [SNOMEDCT_US](https://www.nlm.nih.gov/healthit/snomedct/us_edition.html) codes. 
- This list will be expanded as clinical data transactions, not just metadata, are added for donors for which data is available.
- Similarly, other metadata will be encoded with applicable ontologies.
- The HuBMAP Knowledge Graph underpins all ontologies used in HuBMAP but is not yet deployed.
- The current CCF ontology uses Uberon, Kidney Tissue Atlas Ontology (KTAO) and Cell Ontology (CL), see details in https://arxiv.org/abs/2007.14474 

</details>

## API (Application Programming Interface) Questions

<details><summary><b>Do you support an API for programmatic access?</b> </summary>

The HuBMAP portal is built using an extensible API structure that supports all component interactions. 
- APIs are being registered in [SmartAPI](https://smart-api.info/registry?q=hubmap).
- For external access to APIs, please submit a request to the <a href="mailto:help@hubmapconsortium.org">HuBMAP Helpdesk</a>.

</details>

<details><summary><b>What are the current features of the API?</b> </summary>

The HuBMAP APIs underpin all provenance, data access, processing, translation, search, and access controls. 
- APIs also report the versions and uptime statuses of all Docker containers that comprise HuBMAP’s microservices orchestration architecture.

</details>

<details><summary><b>What are the expectations about future API features?</b> </summary>

- APIs are extensible and are expected to be expanded progressively.
- The next major set of APIs will deliver the underpinning transactions needed for semantic search.

</details>

<details><summary><b>What are the licensing requirements to use HuBMAP APIs, software, and data?</b> </summary>

- The HuBMAP APIs, SmartAPI, and CCF 3D Reference Object Library (data) are released under [Creative Commons Attribution 4.0 International (CC BY 4.0)](https://creativecommons.org/licenses/by/4.0/).
- Most of HuBMAP's software is licensed under the [MIT License](https://en.wikipedia.org/wiki/MIT_License) or [GPL v3 License.](https://www.gnu.org/licenses/gpl-3.0.en.html)
- Most of HuBMAP's source code can be found in GitHub at https://github.com/hubmapconsortium/.
- A few source code repositories use different open source licensing. You can verify this by viewing the LICENSE file in the respective repository.

</details>

<details><summary><b>How do you plan to handle error fixes and updates? Will I be notified of these? </b></summary>

- You can submit a bug or request a new Data Portal feature through the form at https://hubmapconsortium.org/contact-form/.
- For help with specific issues related to the portal, please contact the <a href="mailto:help@hubmapconsortium.org">HuBMAP Helpdesk</a> and submit a support ticket.
- To be sure you are up-to-date on all HuBMAP news, sign up for the mailing list at https://hubmapconsortium.org/hubmap-mailing-list.

</details>

## Experimental Design & Data Analysis Questions

<details><summary><b>Where can I find information about experimental design associated with HuBMAP data? </b></summary>

An overview of the Information on the experimental design and choice of modalities can be found within this reference:
- Snyder, M.P., Lin, S., Posgai, A. et al. The human body at cellular resolution: the NIH Human Biomolecular Atlas Program. Nature 574, 187–192 (2019). https://doi.org/10.1038/s41586-019-1629-x ([PMC6800388](http://www.ncbi.nlm.nih.gov/pmc/articles/pmc6800388/))

Additional information on experimental design for each modality featured in the portal can be obtained on protocols.io as listed below. Further questions can be directed to the dataset contacts detailed within the portal.

Overview protocols:

#### University of Florida:
- CODEX: https://www.protocols.io/view/hubmap-tmc-florida-zurich-codex-modality-overview-be9pjh5n
- 10x: https://www.protocols.io/view/hubmap-uf-tmc-10x-genomics-scrnaseq-modality-overv-be79jhr6
- Imaging Mass Cytometry: https://www.protocols.io/view/imaging-mass-cytometry-modality-overview-bgatjsen

#### Vanderbilt University:
- Overview: https://www.protocols.io/view/vu-biomolecular-multimodal-imaging-center-biomic-k-bfskjncw

#### UCSD:
- https://www.protocols.io/view/human-kidney-urinary-tract-and-lung-cell-type-mapp-6qpvro1d2vmk/v2

</details>

<details><summary><b>Where can I find information about HuBMAP experimental protocols?</b> </summary>

All published protocols that are used in HuBMAP are available on protocols.io here: https://www.protocols.io/groups/human-biomolecular-atlas-program-hubmap-method-development.

</details>

<details><summary><b>How do you define raw and processed data? </b></summary>

We define <em>raw</em> data as the data that comes directly off of the instrument (e.g. mass spectrometer, microscopy, etc.), while <em>processed </em>data has been transformed in some manner (e.g. normalization, background subtracted, aligned, etc.) and the level of processing is defined by the data state as detailed below. Data states are dependent upon the modality. In general, data state 0 (raw data) and state 1 (processed data) are available on the portal for downloading. 

Microscopy:

| Data State | Description | Example file type |
| --- | --- |  --- |
| 0 | Raw image data: This is the data that comes directly off the instrument without preprocessing. (may not always be included). | CZI, TIFF |
| 1 | Processed data: Can include stitching, thresholding, background subtraction, z-stack alignment, deconvolution | CZI, TIFF, OME-TIFF|
| 2 | Segmentation: Computationally predicted cell (nucleus, cytoplasm) and/or structural boundaries (tubules, ventricles, etc.)   | CSV, TIFF |
| 3 | Annotation (Cells and Structures): Interpretation of microscopy image and/or segmentation in terms of biology (e.g. unhealthy vs healthy, cell-type, function, functional region). | TIFF, PNG |

Mass Spectrometry:

| Data State | Description | Example file type |
| --- | --- |  --- |
| 0 | Raw image data: This is the data that comes directly off the mass spectrometer without preprocessing; sometimes referred to as raw spectral data.| imzML |
| 1 | Processed imaging MS data: Can include peak alignment, intensity normalization, m/z recalibration | CSV, OME-TIFF|

Sequencing:

| Data State | Description | Example file type |
| --- | --- |  --- |
| 0 | Raw data: This is the raw sequence data (unprocessed) generated directly by the sequence instrument in files either with Phred quality scores (fastq). | FASTQ |
| 1 | Aligned data: SAM files contain sequence data that has been aligned to a reference genome and includes chromosome coordinates. BAM files are compressed binary versions of SAM files. The reference genome used is hg38.| SAM, BAM|

</details>

<details><summary><b>In what formats is your data available? </b></summary>

- Imaging based raw or processed data is available as TIFF or OME.TIFF formats
- Segmented imaging data is generated as csv and TIFF formats
- Annotated imaging data is TIFF, PNG, and PDF
- Raw sequence data is provided as fastq and metadata via tsv
- Imaging mass spectrometry raw data is provided as a .d and processed data is imzml, or a csv and a series of ome-tiffs.

</details>

<details><summary><b>Can I get the code that you used to process data?</b> </summary>

All available code can be found on the HuBMAP github page (https://github.com/hubmapconsortium).

</details>

<details><summary><b>Can I find an explanation for the choice of HuBMAP pipelines and algorithms?</b></summary>

- Brief descriptions of the HuBMAP data analysis pipelines are available through the portal at https://portal.hubmapconsortium.org/docs/pipelines.
- All code made available to users can be found on the HuBMAP github page (https://github.com/hubmapconsortium).

</details>

<details><summary><b>Which samples should I consider to be technical replicates?</b> </summary>

Technical replicates are repeated measurements of the same existing sample. As even serial tissue sections represent distinct samples, we do not consider any images of tissues to be technical replicates. 

Technical replicates for sequencing assays would be any sequencing libraries generated from the same sample or aliquot of cells or nuclei.

</details>

<details><summary><b>Which samples should I consider to be biological replicates?</b> </summary>

Biological replicates are datasets from samples that originate from the same organ and organ donor. As such, each dataset within the HuBMAP database that is provided for a given donor organ for a comparable anatomical region/structure would be a biological replicate. 

</details>

<details><summary><b>How are each organ/tissue handled to maintain high reproducibility and validity for the assay? </b></summary>

Protocols.io detailed processing with QA/QC

[Human BioMolecular Atlas Program (HuBMAP) Method Development Community timeline](https://www.protocols.io/workspaces/human-biomolecular-atlas-program-hubmap-method-development)

</details>

<details><summary><b>How have antibodies been verified? </b></summary>

Initially, all antibodies were validated by individual groups. With later data releases, complete antibody information, including antibody clone, vendor, RRID, conjugation information, etc. became available. Additional antibody validation standards were also implemented. For the development of our antibody validation levels, we followed the antibody verification guidelines established in the following manuscripts:

[A proposal for validation of antibodies.](https://pubmed.ncbi.nlm.nih.gov/27595404/)
- Uhlen M, Bandrowski A, Carr S, Edwards A, Ellenberg J, Lundberg E, Rimm DL, Rodriguez H, Hiltke T, Snyder M, Yamamoto T. Nat Methods. 2016 Oct;13(10):823-7. doi: 10.1038/nmeth.3995. Epub 2016 Sep 5.PMID: 27595404

[The Antibody Society’s antibody validation webinar series](https://pubmed.ncbi.nlm.nih.gov/32748696/)
- Voskuil, J., Bandrowski, A., Begley, C. G., Bradbury, A., Chalmers, A. D., Gomes, A. V., Hardcastle, T., Lund-Johansen, F., Plückthun, A., Roncador, G., Solache, A., Taussig, M. J., Trimmer, J. S., Williams, C., & Goodman, S. L. MAbs. 2020;12(1):1794421. doi:10.1080/19420862.2020.1794421. PMID: 32748696

</details>
